custom_dc/prepare_data.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Prepare Data
+nav_order: 2
+parent: Custom Data Commons
+published: true
+---
+
+## Overview
+
+Preparing data involves cleaning / formatting the raw data into compatible CSV
+file. The CSV file is expected to have columns corresponding to the Values
+(numeric) about a Variable, Place and Date. The format of a CSV file is
+specified by a [Template
+MCF](https://github.com/datacommonsorg/data/blob/master/docs/mcf_format.md#template-mcf).
+The ready to use artifacts contains one TMCF file (_.tmcf) and a few compatible
+CSV files (_.csv).
+
+## General CSV format
+
+In the table shown below, there are separate columns for Variable (Variable),
+Place (Country), Date (Year) and Value (Value) and each row of the CSV
+corresponds to one observation of the Variable about a Place at the specified
+Date.
+
+| Year | Country | Variable        | Value       | Extra Column |
+| ---- | ------- | --------------- | ----------- | ------------ |
+| 2017 | UK      | Life_Expectancy | 81.25609756 | 1            |
+| 2017 | UK      | Population      | 65844142    | 2            |
+
+The TMCF for this CSV looks like:
+
+```txt
+Node: E:CSV_FILE_NAME->E0
```

Comment:
Also add a note about the format here? CSV_FILE_NAME might be used verbatim from the example.

Reply:
changed to data as it does not matter or need to match file name